src/ice_model.F90

```diff
@@ -1269,6 +1269,10 @@ subroutine set_fast_ocean_sfc_properties( Atmos_boundary, Ice, IST, Rad, FIA, &
     Ice%t_surf(i2,j2,k2) = US%C_to_degC*Rad%t_skin(i,j,k) + T_0degC
   enddo ; enddo ; enddo
 
+  ! coszen has changed if we are using the specified ice mode
+  if (associated(Ice%sCS)) then
```

Comment:
@marshallward How about this? Does it pass muster?

Reply:
Thank you Niki, this looks good, approving it now.